README.md

```diff
@@ -180,6 +210,5 @@ public class TestTiDBTableSource {
 
 ## TODO
 
-1. Upgrade flink version to 1.11;
-2. Support write data in Presto-TiDB-Connector;
-3. TiDBTableSink.
\ No newline at end of file
+1. Support write data in Presto-TiDB-Connector;
```

Comment:
Support writing data in Presto-TiDB-Connector;

Reply:
This is my carelessness, and I have fixed it in the next commit